### PR TITLE
Skip agent 000 in GET /agents/summary/os-status endpoints

### PR DESCRIPTION
**Pull request by mcarmona99**
|Related issue|
|---|
| #6502 |

Hi team,

In this PR, I have changed the `GET /agents/summary/os` and `GET /agents/summary/status` endpoints behavior in order to not consider the agent `000`. Not considering `000` will fix the error reported in the issue related.

- Testing that the endpoints are not returning 000 information:

*GET /agents/summary/status:*

Response before changes:

```
{
  "data": {
    "active": 9,
    "disconnected": 0,
    "never_connected": 4,
    "pending": 0,
    "total": 13
  },
  "error": 0
}
```

Response after changes:

```
{
  "data": {
    "active": 8,
    "disconnected": 0,
    "never_connected": 4,
    "pending": 0,
    "total": 12
  },
  "error": 0
}
```

*GET /agents/summary/os:*

I have deleted all agents so the only agent we have is 000.

```
sqlite> select os_platform,id from agent;
ubuntu|0
```

Endpoint response: 

```
{
  "data": {
    "affected_items": [],
    "total_affected_items": 0,
    "total_failed_items": 0,
    "failed_items": []
  },
  "message": "Could not get the operative system of the agents",
  "error": 0
}
```

- API integration tests:

```
test_agents_GET_endpoints.tavern.yaml 
	 90 passed, 97 warnings
	 
test_rbac_black_agents_endpoints.tavern.yaml 
	 39 passed, 41 warnings

test_rbac_white_agents_endpoints.tavern.yaml 
	 39 passed, 41 warnings
```

- Unit tests:

```
framework/wazuh/core/tests/test_agent.py
======================= 187 passed, 11 warnings in 3.33s =======================

framework/wazuh/tests/test_agent.py
======================= 89 passed, 11 warnings in 13.56s =======================
```

Regards,
Manuel.

### CODE REVIEW

File: framework/wazuh/agent.py [hunk 1]
```diff
@@ -62,12 +62,23 @@ def get_distinct_agents(agent_list=None, offset=0, limit=common.database_limit,
...
+    Parameters
+    ----------
+    agent_list : list[str]
+       List of agents ID's
+
+    Returns
+    -------
+    WazuhResult
     """
+    # We don't consider agent 000 in order to get the summary
```
Comment (davidjiglesias) [inline framework/wazuh/agent.py:74]: It is much more efficient and clean to just send the WazuhDBQueryAgents with q='id!=000', like in https://github.com/wazuh/wazuh/blob/fad97a057deb37dc5c806c0d5e39038de2aa76ad/framework/wazuh/agent.py#L861

File: framework/wazuh/agent.py [hunk 2]
```diff
@@ -83,14 +94,25 @@ def get_agents_summary_status(agent_list=None):
...
```
Comment (davidjiglesias) [inline framework/wazuh/agent.py:106]: It is much more efficient and clean to just send the WazuhDBQueryAgents with q='id!=000', like in https://github.com/wazuh/wazuh/blob/fad97a057deb37dc5c806c0d5e39038de2aa76ad/framework/wazuh/agent.py#L861

**Review by davidjiglesias — CHANGES_REQUESTED** [review]
We also need to change code and tests for overview endpoint to not include 000.
Update changelog.

**Comment by mcarmona99** [comment]
- Now the endpoint `GET /overview/agents` skip the agent with id `000`. Changed the endpoint docstring to NumpyDoc.

- Update `CHANGELOG.md`. 

- API integration test results:

```
test_overview_endpoints.tavern.yaml 
	 1 passed, 3 warnings

test_rbac_black_overview_endpoints.tavern.yaml 
	 1 passed, 3 warnings

test_rbac_white_overview_endpoints.tavern.yaml 
	 1 passed, 3 warnings

```

**Review by davidjiglesias — APPROVED** [review]

**Pull request merged**